docs/ml-datasource.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: global
+title: Data sources
+displayTitle: Data sources
+---
+
+In this section, we introduce how to use data source in ML to load data.
+Beside some general data sources "parquat", "csv", "json", "jdbc", we also provide some specific data source for ML.
```

Comment:
- data sources -> data sources like
- Do you want to just say "Parquet, CSV, JSON, and JDBC"? they aren't code identifiers here
- parquat -> parquet
- data source -> data sources